src/object_properties.h

```diff
@@ -60,6 +59,7 @@ struct ObjectProperties
 	std::string wield_item;
 	bool static_save = true;
 	float eye_height = 1.625f;
+	float zoom_fov = 15.0f;
```

Comment:
This means the server will force a zoom fov of 15 as soon the ObjectProperties are sent to the server. The client has no control about the limits.

Reply:
So get the setting here too i guess.
In testing there wasn't an issue though, but i'll try it.

Reply:
Actually this struct does not set the defaults for properties, that's done elsewhere, these values are overriden. Good point though, i did wonder about this and investigate it, and this is why this line doesn't get the client setting. Testing showed no issues either.

Reply:
But i will investigate more and test again.

Reply:
But see my other line comments for a related problem.